app/editor/src/components/form/wysiwyg/Wysiwyg.tsx

```diff
@@ -76,16 +84,16 @@ export const Wysiwyg: React.FC<IWysiwygProps> = ({ fieldName, label, required, e
 
   const stripHtml = () => {
     // strip html from string
-    let doc = new DOMParser().parseFromString(values[fieldName] as string, 'text/html');
-    setFieldValue(fieldName, doc.body.textContent || '');
+    let doc = new DOMParser().parseFromString(values[fieldName!] as string, 'text/html');
```

Comment:
This will fail for the same reason.  And any other instance where you have `fieldName!`.  The exclamation mark is simply a way to ignore the lint error.  It will cause run time errors.

Reply:
Updated with checking code, thanks